Formula/scummc.rb

```diff
@@ -0,0 +1,38 @@
+class Scummc < Formula
+  desc "Set of tools allowing to create SCUMM games"
+  homepage "https://github.com/AlbanBedel/scummc#readme"
+  url "https://github.com/AlbanBedel/scummc/archive/v0.2.1.tar.gz"
+  sha256 "1d9b412d7ab6197495589a19a4745f58b35a002e26cea21bf6874a7ae4f1fd2e"
+
+  depends_on "bison" => :build
+  depends_on "freetype" => :build
+  depends_on "gtk+" => :build
+  depends_on "readline" => :build
+  depends_on "sdl" => :build
```

Comment:
Is this doing a static build? Asking because `sdl` being a `:build`-only dep is incredibly rare.

Reply:
Poking at the history I see Misty already asked and you're unsure. I suspect if we add a meaningful test here we'll quickly find out.